app/pages/analyst/application/[applicationId]/assessments/screening.tsx

```diff
@@ -40,7 +58,13 @@ const ScreeningAssessment = ({
     <Layout session={session} title="Connecting Communities BC">
       <AnalystLayout query={query}>
         <AssessmentsTabs />
-        <GuideLink />
+        <StyledInfoBarDiv>
+          <GuideLink />
+          <StyledLink href={CCBC_ELIGIBILITY_SCREENING_TEMPLATE}>
+            <FontAwesomeIcon icon={faFileLines} /> GuideCCBC_Eligibility
```

Comment:
I think the merge put in an extra `Guide` here

Reply:
yeah, removed